integration-tests/src/test/java/org/apache/druid/tests/indexer/ITIndexerTest.java

```diff
@@ -34,16 +34,25 @@
   private static final String INDEX_QUERIES_RESOURCE = "/indexer/wikipedia_index_queries.json";
   private static final String INDEX_DATASOURCE = "wikipedia_index_test";
 
+  private static final String INDEX_WITH_TIMESTAMP_TASK = "/indexer/wikipedia_with_timestamp_index_task.json";
+  // TODO: add queries that validate timestamp is different from the __time column since it is a dimension
```

Comment:
Would you open an issue for this instead of TODO?

Reply:
Done